python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -563,32 +564,32 @@ def default_description_fn(dbt_resource_props: Mapping[str, Any], display_raw_sq
 
 def default_asset_check_fn(
     manifest: Mapping[str, Any],
-    dbt_nodes: Mapping[str, Any],
     dagster_dbt_translator: "DagsterDbtTranslator",
     asset_key: AssetKey,
     test_unique_id: str,
+    project: Optional[DbtProject],
```

Comment:
I believe changing the signature of this might be a breaking change? I remember that we were recommending using these functions at some point, when customizing the translator methods. I don't think it's part of our documentation anymore, but wondering if that could break user code?

Reply:
Maybe we could deprecate the `dbt_nodes` parameter and set the default to None for `project`?

I think these changes are fairly low risk, but just to make sure we don't break user code.

Reply:
hm in what cases do you think a user would use this default fn? it only gets called in `build_dbt_specs` not the translator and its not exported from the top